HumanEvalLean/HumanEval51.lean

```diff
@@ -1,5 +1,48 @@
-def remove_vowels : Unit :=
-  ()
+import Std.Data.Iterators.Combinators.FilterMap
+
+def is_subseq (s₁ : String) (s₂ : String) : Prop :=
```

Comment:
Would you mind adapting the capticalization of your names to the lean4 convention, i.e., `UpperCamelCase` for types/propositions, `lowerCamelCase` for definitions and mixed case for theorems? The names according to the convention should be

- `IsSubseq`
- `vowels`
- `MaximalFor`
- `RemoveVowelsIff`
- `removeVowels`
- `removeVowels_correct`

Reply:
Done in ab70eb23d90544dd34ed19ee339381ddd9fffbed